opentelemetry-stdout/examples/basic.rs

```diff
@@ -28,7 +28,7 @@ static RESOURCE: Lazy<Resource> = Lazy::new(|| {
 fn init_trace() {
     let exporter = opentelemetry_stdout::SpanExporter::default();
     let provider = TracerProvider::builder()
-        .with_simple_exporter(exporter)
+        .with_batch_exporter(exporter, runtime::Tokio)
```

Comment:
lets keep it simple itself. stdout is only for learning purposes. and simple exporter is also only meant for that.